DTS-STN/canadian-dental-care-plan
# AB#2858 header dropdown with Logout button

### Description
Added Drop-down menu for the header.

It is simplified, but most of the code was copied from SCCH.

### Related Azure Boards Work Items
https://dev.azure.com/dts-stn/canada%20dental%20care%20plan/_backlogs/[AB#2858](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2858)


### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/77116011/6c7fc5ac-f524-4af3-b4bf-df8fdfaf9fba)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/77116011/34cc4da9-98bf-4e88-8bb7-b9b968052914)


### Checklist

- [x] I have checked that all e2e tests pass by running `npm run test:e2e -- run`

### Test Instructions
1. Run localy `npm run dev`
2. Go on any page past language selection.
3. Dropdown should be in the header and logout redirects to `/auth/logout`

### Additional Notes
Still new with TypeScript.
